README.md

```diff
@@ -26,7 +26,7 @@ more information, consult [the docs](https://docs.getdbt.com/docs/profile-spark)
 
 ## Running locally
 A `docker-compose` environment starts a Spark Thrift server and a Postgres database as a Hive Metastore backend.
-Note that this is spark 2 not spark 3 so some functionalities might not be available.
+Note: Spark has moved to Spark 3 (formerly on Spark 2).
```

Comment:
you mean dbt / dbt-spark moved to spark-3 right?

Reply:
I thought about this statement and I'm a bit confused. 
If `dbt-spark` moved to 3, the testing module still is on 2 and this line in the docs doesn't add up. 
Since this is a Draft PR, I'll finish up the testing and clean this up in either this PR or a separate one.
But @rvacaru do you know the context behind why the compose file has spark:3 but the testing didn't move over ?